scripts/start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Does this need to be bash?  set -u is POSIX, isn't it?

Reply:
Good catch.